composer.json

```diff
@@ -28,7 +28,7 @@
     "require": {
         "php": "^7.1",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
-        "slevomat/coding-standard": "^5.0",
+        "slevomat/coding-standard": "dev-master#63a8186b129ee96d1277e68c80cf87c8cdb356d1",
```

Comment:
@malarzm @deeky666 I'm likely raising a separate PR for stable slevomat release pinning by @kukulich.

Reply:
IIRC pinning a commit could lead to weird side effects where we can end up with vendor requirements from current `dev-master` while code in pinned commit could require something else.

Reply:
Yes: this is not for a stable release yet. I'd need a SemVer selector for the release to be tagged.